clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -4005,6 +4005,13 @@ def SinOp : UnaryFPToFPBuiltinOp<"sin", "SinOp">;
 def SqrtOp : UnaryFPToFPBuiltinOp<"sqrt", "SqrtOp">;
 def TruncOp : UnaryFPToFPBuiltinOp<"trunc", "FTruncOp">;
 
+def AbsOp : CIR_Op<"abs", [Pure, SameOperandsAndResultType]> {
+  let arguments = (ins CIR_IntType:$src, I1Attr:$is_int_min_poison);
```

Comment:
For attributes that behave like a boolean flag, we prefer `UnitAttr` instead of `I1Attr`.

Reply:
Please just rename this to `poison` and add some documentation (see other operations) to explain what poison means in this context.